Documentation/gettingstarted/kube-router.rst

```diff
@@ -140,4 +140,4 @@ installed:
   * ``10.2.2.0/24 dev tun-172011760 proto 17 src 172.0.50.227``
   * ``10.2.3.0/24 dev tun-1720186231 proto 17 src 172.0.50.227``
 
-.. include:: k8s-install-connectivity-test.rst
+.. include:: k8s-install-validate.rst
```

Comment:
To reviewers: will `kube-router` allow for usage of Cilium CLI? If not, then we should revert to manually linking to
```suggestion
.. include:: k8s-install-connectivity-test.rst
```